app/timetables/models.py

```diff
@@ -24,6 +24,16 @@ class Weekday(SlugifyMixin, models.Model):
     def __str__(self):
         return self.name
 
+    def clean(self):
+        # Call it's super clean method first so slug field would have
+        # been created from Slugify Mixin's clean method before we do our check
+        super().clean()
+        # Ensure that weekday name entered is a valid one
+        weekdays = ['monday', 'tuesday', 'wednesday', 'thursday',
```

Comment:
Move this variable out of the clean method so that it can be used outside the model whenever week days are needed.

Add it as a model constant `WEEK_DAYS`